### DISCUSSION TITLE
mapping-sensors-output-to-physical-drives-in-linux/

### DISCUSSION THREAD

**Discussion by giscus** [ironicbadger/blog.ktz.me#220] [Announcements]
# mapping-sensors-output-to-physical-drives-in-linux/

Whilst building out some new Grafana dashboards over the weekend I made the switch from InfluxDB to Promtheus node exporters for scraping metrics from Linux servers. But how do you marry up the drivetemp-scsi-0-10 value output from sensors to a physical disk?

https://blog.ktz.me/mapping-sensors-output-to-physical-drives-in-linux/

**Comment by frame45** [comment]
Question, how do I get debian to show the "drivetemp-scsi-*-*" data, it's not detecting a SCSI or SATA adapter. I ran "sensors-detect" and restarted kmod, but still have nothing. Am I missing a step here?

**Comment by Charlotte-br560** [comment]
Thanks for sharing your experience with transitioning to Prometheus node exporters. Mapping sensors output to physical drives in Linux can indeed be a puzzle. I found this article super helpful in tackling that exact challenge: Mapping Sensors Output to Physical Drives in Linux.
